### PR TITLE
Require qpid and dispatch-router on EL8 and EL7

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: packages/katello/katello/katello.spec [hunk 1]
```diff
@@ -36,6 +36,10 @@ Requires: %{?scl_prefix}rubygem-hammer_cli
 Requires: %{?scl_prefix}rubygem-hammer_cli_foreman
 Requires: %{?scl_prefix}rubygem-hammer_cli_katello
 
+# katello-agent requirements
+Requires: qpid-cpp-server-linearstore
+Requires: qpid-cpp-client-devel
```
Comment (jlsherrill) [inline packages/katello/katello/katello.spec:41]: is client-devel actaully a run time requirement?

File: packages/katello/katello/katello.spec [hunk 2]
```diff
@@ -113,11 +114,17 @@ install -m 644 ./manpages/katello-change-hostname.8.gz %{buildroot}/%{_mandir}/m
...
 Requires:       rubygem-highline
 Requires:       %{name}-debug
 %if 0%{?rhel} == 8
 Requires:       container-selinux
 %endif
+=======
+Requires: rubygem-highline
+Requires: %{name}-debug
+Requires: qpid-dispatch-router
+>>>>>>> 6ddc6d02c (Require qpid and dispatch-router on EL8 and EL7)
```
Comment (jlsherrill) [inline packages/katello/katello/katello.spec:127]: image

File: packages/katello/katello/katello.spec (hunk 1 above)
Comment (ehelms) [reply packages/katello/katello/katello.spec:41]: Thats a good question

Comment (jlsherrill) [reply packages/katello/katello/katello.spec:41]: oh i see this is actually down below, is it actually required?

Comment (ehelms) [reply packages/katello/katello/katello.spec]: One way to find out, removed.

**Review by jlsherrill — APPROVED** [review]

**Pull request merged**